UnitTestFrameworkPkg/Library/UnitTestPersistenceLibSimpleFileSystem/UnitTestPersistenceLibSimpleFileSystem.c

```diff
@@ -31,7 +31,8 @@
 STATIC
 EFI_DEVICE_PATH_PROTOCOL *
 GetCacheFileDevicePath (

```

Comment:
It doesn't look like the return value is actually used in the file now. Is it needed? Can it just return the name?

Reply:
I was thinking about this, too. How about just change the function (and its name) to return file name only?

Reply:
I'm in favor of that.

Reply:
Updated to return file name and path instead of device path.